scene/audio/audio_stream_player.cpp

```diff
@@ -103,7 +103,7 @@ void AudioStreamPlayer::_mix_audio() {
 		if (stream_stop) {
 			stream_playback->stop();
 			active = false;
-			set_process_internal(false);
```

Comment:
There is not any point really in using physics process for this

Reply:
I have difficulty understanding why this solves our problem as well, but it does.
Without this we get `ERROR: add_to_group: Already in group: idle_process_internal` (as well as unpredictable behavior for anything using idle_process that is after the AudioStreamPlayer in the scene tree) and with it we don't.

The reason why we tried this approach is that in the AudioStreamPlayer2D and AudioStreamPlayer3D we are using physics process. If AudioStreamPlayer is indeed not intended to mix in the physics process then there is an error somewhere else that i cannot find.

Reply:
I understand this may make a bug not appear, but best is to still attempt to create a reproduction case which shows the original bug, as this may be just a workaround that does not fix anything.

Reply:
We are having difficulties creating a reproduction case, but we will give it another try then.

Reply:
See #27590 for reproduction project and hopefully a much better description of the problem.